kuwala/core/canvas/src/components/Header.js

```diff
@@ -102,6 +103,9 @@ const Header = () => {
                     <div className="relative">
                         <button
                             className="p-2 bg-gray-100 rounded-full hover:bg-gray-200 focus:outline-none focus:ring"
+                            onClick={() => {
```

Comment:
Please remove these changes as they are not functional. We should use a debugger and not introduce a console log. ;)

Reply:
removed, sorry lol